src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.Unix.cs

```diff
@@ -7,6 +7,20 @@ public partial class ZipArchiveEntry
     {
         internal const ZipVersionMadeByPlatform CurrentZipPlatform = ZipVersionMadeByPlatform.Unix;
 
+        /// <summary>
+        /// The default external file attributes are used to support zip archives on multiple platforms.
+        /// For Unix the default external attributes are an equivalent of (Convert.ToInt32("644", 8) &lt;&lt; 16)
+        /// where 644 are rw-r--r-- unix file permissions.
+        /// </summary>
+        internal const uint DefaultFileExternalAttributes = 420 << 16;
+
+        /// <summary>
+        /// The default external directory attributes are used to support zip archives on multiple platforms.
+        /// For Unix the default external attributes are an equivalent of (Convert.ToInt32("755", 8) &lt;&lt; 16)
+        /// where 755 are rwxr-xr-x unix file permissions.
+        /// </summary>
+        internal const uint DefaultDirectoryExternalAttributes = 493 << 16;
```

Comment:
@carlossanlop similar to the change here, we probably need a default for directories also in the Tar implementation.